modules/openy_gc_personal_training/README.md

```diff
@@ -5,6 +5,11 @@ This module provides base implementation of Virtual Y 1on1 Meetings.
 ![Virtual Y 1on1 meeting](assets/vy_14_virtual_meeting.png "Virtual Y 1on1 Meeting Demo")
 Short video demo: https://www.loom.com/share/c49ef7bb4b2a4435ac78bfc53582e2f1
 
+## Requirements
+
+This module requires [State Machine](drupal.org/project/state_machine) contrib project. It is absent in the Virtual Y composer.json since we dont want to install unneeded modules to all VY projects.
```

Comment:
> It is absent in the Virtual Y composer.json since we dont want to install unneeded modules to all VY projects.

Adding to composer.json would only add it to the codebase, then if it was declared as a dependency in `openy_gc_personal_training.info.yml` it would only get installed when the Personal Training module is installed.

Adding a requirement but making admins jump through hoops seems to me like more trouble than just adding the module to `composer.json`.